docker-compose.yml

```diff
@@ -2,20 +2,93 @@
 # https://support.circleci.com/hc/en-us/articles/360007324514-How-can-I-use-Docker-volume-mounting-on-CircleCI-
 # For this reason, we are creating specific dockerfiles, and building the images from them, copying the files in the images
 # instead of mounting them.
+
+
 version: "3.3"
 services:
-  stardog:
+  zoo1:
+      image: zookeeper:${ZOO_TAG}
+      container_name: zkn1
+      environment:
+          ZOO_MY_ID: 1
+          ZOO_SERVERS: server.1=0.0.0.0:2888:3888 server.2=zoo2:2888:3888 server.3=zoo3:2888:3888
+
+  zoo2:
```

Comment:
Why you running 3 zoo keepers? you really only need one to test the cluster operations and that will simplify startup and wait times. We don't need the redundancy as we aren't testing the cluster, just accessing it works.